pulp_service/pulp_service/app/authentication.py

```diff
@@ -5,3 +5,6 @@ class RHServiceAccountCertAuthentication(JSONHeaderRemoteAuthentication):
 
     header = "HTTP_X_RH_IDENTITY"
     jq_filter = ".identity.x509.subject_dn"
+
+    def authenticate_header(self, request):
+        return self.__name__
```

Comment:
This produce a run time error: AttributeError: 'RHServiceAccountCertAuthentication' object has no attribute '__name__'

Can you just change this to "Bearer" string?